SignalRServiceBenchmarkPlugin/framework/master/Program.cs

```diff
@@ -70,19 +74,8 @@ private static async Task InstallPluginInSlaves(IList<IRpcClient> clients, strin
             var tasks = new List<Task<bool>>();
 
             // Try to install plugin
-            foreach (var client in clients)
-            {
-                tasks.Add(client.InstallPluginAsync(moduleName));
-            }
-            await Task.WhenAll(tasks);
-
-            // Check whether the plugin installed
-            var success = true;
-            foreach (var task in tasks)
-            {
-                if (!task.Result) success = false;
-                break;
-            }
+            var installResults = await Task.WhenAll(from client in clients select client.InstallPluginAsync(moduleName));
+            var success = installResults.All(result => result == true);
```

Comment:
simplify: `var success = installResults.All(x => x);`

Reply:
Got it, thanks